<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderDoubleCombo</class>
 <widget class="QWidget" name="SliderDoubleCombo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>50</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SliderCombo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="base_" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#base_
{
  border-radius: 3px;
  border: 1px solid rgba(0,0,0,30);
  background-color: qlineargradient(spread:pad, x1:0.475, y1:0, x2:0.486, y2:1, stop:0 rgba(0, 0, 0, 10), stop:0.276836 rgba(255, 255, 255, 0), stop:1 rgba(0, 0, 0,20));
}

QSlider#horizontalSlider
{
  background-color: rgba(255, 255, 255, 0);
}

QWidget#widget_2
{
  background-color: rgba(255, 255, 255, 0);
}

QPushButton#edit_button_
{
  background-color: rgba(255, 255, 255, 0);
  border-color: rgba(255, 255, 255, 0);
  
  font: 7pt &quot;Arial&quot;;
}

QPushButton#edit_button_:checked
{
  background-color: rgba(255, 255, 255, 0);
  border: 1px solid rgba(255, 255, 255, 0);
  
  font: bold 8pt &quot;Arial&quot;;
  color: rgb(85, 170, 255);
}

QPushButton#edit_button_:pressed
{
  background-color: rgba(255, 255, 255, 0);
  border: 1px solid rgba(255, 255, 255, 0);
  
  font: bold 8pt &quot;Arial&quot;;
  color: rgb(85, 170, 255);
}

QPushButton#increase_range_button_
{
  margin: 0px;
  padding: 0px;
}

QPushButton#decrease_range_button_
{
  margin: 0px;
  padding: 0px;
}

QWidget#widget_3{ border: 1px solid rgb(240, 240, 240); border-radius: 3px;}

</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="range_" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,1">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="min_">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="decrease_range_button_">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Half the current range</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_button_">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>7</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="toolTip">
               <string>Click here to enable range editing</string>
              </property>
              <property name="text">
               <string>edit range</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="increase_range_button_">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Double the current range</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="max_">
              <property name="text">
               <string>100</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="spinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
